### PR TITLE
Use correct signing identities for each target and configuration

### PR DESCRIPTION
**Pull request by akolov**

### CODE REVIEW

File: Cartography.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -788,6 +788,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				APPLICATION_EXTENSION_API_ONLY = YES;
+				CODE_SIGN_IDENTITY = "Mac Developer";
```
Comment (robb) [inline Cartography.xcodeproj/project.pbxproj]: What issue does this fix for you?

Comment (akolov) [inline Cartography.xcodeproj/project.pbxproj]: Our CI doesn't have development identities installed, and fails to build at codesign step if we leave it at is.
I'm building on Bitrise with Carthage.

Comment (robb) [inline Cartography.xcodeproj/project.pbxproj]: But this changes it from doing no codesigning to signing with the _Mac Developer_ identity, right?

Btw, if you're only dealing with iOS, you can use `carthage build --platform ios` to skip building for Mac altogether.

Comment (akolov) [inline Cartography.xcodeproj/project.pbxproj]: No, originally you had "iPhone Developer" identity in the global project setting, which is then propagated to Mac Framework as well.
As for the platform setting, I'm aware of it, but thanks :)

**Pull request merged**